interface/main/main_screen.php

```diff
@@ -116,6 +116,10 @@
 if ($GLOBALS['new_tabs_layout']) {
```

Comment:
what do you think about using the following here:
``````
// For app mode, always use tabs layout
if ( $GLOBALS['new_tabs_layout'] || ((!empty($_POST['appChoice'])) && ($_POST['appChoice'] !== '*OpenEMR')) ) {